src/read.rs

```diff
@@ -399,18 +427,18 @@ impl<R: Read + io::Seek> ZipArchive<R> {
     ///
     /// This uses the central directory record of the ZIP file, and ignores local file headers
     pub fn new(mut reader: R) -> ZipResult<ZipArchive<R>> {
-        let (footer, cde_start_pos) = spec::CentralDirectoryEnd::find_and_parse(&mut reader)?;
+        let (footer, cde_end_pos) = spec::CentralDirectoryEnd::find_and_parse(&mut reader)?;
```

Comment:
Revert this name change. CDE stands for Central Directory End, and cde_start_pos is the start of the CDE record.